r-p-s.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="r-p-s.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>Rock, Paper, Scissors Game</h1>
+
+    <p>
+        <label><input type="radio" name="game" value="rock" checked>Rock<br></label>
+        <label><input type="radio" name="game" value="paper">Paper<br></label>
+        <label><input type="radio" name="game" value="Scissors">Scissors<br></label>
+
+    </p>
+    <div>
+        <button id="selection">Play</button>
+        <button id="reset">Reset</button>
+    </div>
+    <p>
```

Comment:
I would change this to look something like 
`<p>Rules</p>
<ul>
            <li>paper beats rock,</li>
            <li>rock beats scissors,</li>
            <li>scissors beats papers</li>
</ul>`